backend/test_observer/data_access/models.py

```diff
@@ -402,3 +406,41 @@ def __repr__(self) -> str:
             "test_execution_id",
             "test_case_id",
         )
+
+
+class TestStatusUpdate(Base):
+    """
+    A table to represent status updates for the test runs
+    """
+
+    __tablename__ = "test_status_update"
+
+    agent_id: Mapped[str]
+    job_queue: Mapped[str]
+    events: Mapped[list["TestEvent"]] = relationship(
+        back_populates="test_status_update", cascade="all, delete"
+    )
```

Comment:
it would be useful to have the events relationship order by timestamp by default